roles/sft-server/molecule/without-metrics/converge.yml

```diff
@@ -0,0 +1,28 @@
+- name: Provision SFT Server
+  hosts: all
+  tasks:
+    - name: check INSTANCE_UUID is set
+      assert:
+        that:
+          - molecule_instance_id != ""
+        msg: >
+          [ERR] Please run:
+            export INSTANCE_UUID=$RANDOM
+          then re-run this playbook.
+
+    # Normally we assume DNS to be created externally-to-ansible.
+    # Here, we create a DNS record using ansible only for molecule tests.
+    - name: A records
+      route53:
+        command:   create
+        zone:      "{{ sft_root_domain }}"
+        record:    "{{ sft_fqdn }}"
+        type:      A
+        ttl:       60
+        value:     "{{ hostvars[groups['all'][0]].ansible_default_ipv4.address }}"
+        overwrite: yes
+      delegate_to: localhost
+
+    - name: "Statically import sft-server role"
+      import_role:
+        name: "sft-server"
```

Comment:
Having two separate converge files for the two scenarios is fine, but then I would suggest to keep the one from the default scenario inside the default folder. (It's now in the "shared" folder but only used by one scenario)
Alternatively, you could branch with a `when` statement on the same variable (expose metrics) for the difference:

```diff
git diff --no-index molecule/without-metrics/converge.yml molecule/converge.yml 
diff --git a/molecule/without-metrics/converge.yml b/molecule/converge.yml
index 3a6d857..435ceb9 100644
--- a/molecule/without-metrics/converge.yml
+++ b/molecule/converge.yml
@@ -1,3 +1,11 @@
+---
+- name: Create Certs
+  hosts: localhost
+  tasks:
+    - name: Include monitoring-certs
+      import_role:
+        name: "sft-monitoring-certs"
+
 - name: Provision SFT Server
   hosts: all
   tasks:
@@ -26,3 +34,6 @@
     - name: "Statically import sft-server role"
       import_role:
         name: "sft-server"
+    - name: "Statically import srv-announcer role"
+      import_role:
+        name: "srv-announcer"
```

Reply:
good point, I moved it back in.